webpack.config.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+module.exports = {
+  entry: {
+    app: './src/index.js',
+  },
+  output: {
+    path: path.resolve(__dirname, 'build'),
+    filename: '[name].bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+          },
+        },
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader',
+        ],
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'images/[name]-[hash].[ext]',
+            },
+          },
+        ],
+      },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'fonts/[name]-[hash].[ext]',
+            },
+          },
+        ],
+      },
+    ],
+  },
+  plugins: [
+    new CleanWebpackPlugin(['build']),
```

Comment:
Are these executed in order? Are there any order concerns here at all?

Reply:
I believe the answer is yes, order matters. This [SO answer](https://stackoverflow.com/a/41473106) indicates the same, however, I think most plugins operate off events so this doesn't matter _too much_.

In any event, with order as a concern, having `CleanWebpackPlugin` fire first is really the thing that matters. We don't want it to fire later which would remove the files added by the two subsequent plugins. I am going to play around with this to see if this would really happen if we bumped `CleanWebpackPlugin` to last in this list.

Reply:
Comment with any findings you have from testing, please!

Reply:
Testing confirms that order doesn't matter. I put `CleanWebpackPlugin` at the end and it did not wipe the build directory. 👍